components/gitpod-protocol/src/messaging/error.ts

```diff
@@ -140,6 +140,9 @@ export const ErrorCodes = {
     // 501 EE Feature
     EE_FEATURE: 501 as const,
 
+    // 501 Not Implemented
+    NOT_IMPLEMENTED: 501 as const,
```

Comment:
we should not add new error codes, why do we need it?

Reply:
Will it make sense to have someone define `proto` file then other's to implement it? Then we may need NOT_IMPLEMENTED error, but yes, I should remove it since it's not used in this PR